dist/example-config.js

```diff
@@ -193,12 +181,24 @@ var config = {
           }
         }
       },
-      dataSource: {
-        request: {
-          url: '//parsnip.msapp.co.nz/{x}/{y}'
-        },
-        type: 'singlePoll'
-      }
+      dataSource: 'parsnip'
+    }
+  ],
+  dataSources: [
+    {
+      name: 'geojson-spew',
```

Comment:
Only that we aren't currently needing to access the dataSource config anywhere other than the building of the DataServices, where we loop over the config objects creating a DataService object from each config objects. Having them as a keyed object just changes the iteration method I guess, but semantically it makes sense to me that there are a list (array) of dataSources.